### PR TITLE
Add reference image shortcuts

### PR DESCRIPTION
**Pull request by Qainguin**
Adds shortcuts for actions relating to reference images

- Load reference image
- Show reference image
- Overlay reference image

### CODE REVIEW

File: src/GlobalSettings.gd [hunk 1]
```diff
@@ -93,6 +94,8 @@ const keybinds_dict = {
 		"view_show_grid": true,
 		"view_show_handles": true,
 		"view_rasterized_svg": true,
+		"view_reference_image": true,
+		"view_overlay_reference": true
```
Comment (MewPurPur) [inline src/GlobalSettings.gd:98]: Add a trailing comma on the last line as per GDScript's style guide.

File: src/TranslationUtils.gd [hunk 2]
```diff
@@ -3,7 +3,8 @@ class_name TranslationUtils extends RefCounted
 static func get_shortcut_description(action_name: String) -> String:
 	match action_name:
 		"export": return TranslationServer.translate("Export")
-		"import": return TranslationServer.translate("Import")
+		"import": return TranslationServer.translate("Import SVG")
+		"import_reference_image": return TranslationServer.translate("Import Reference Image")
```
Comment (MewPurPur) [inline src/TranslationUtils.gd:7]: I'd suggest using "Load reference image" without modifying the "Import" string.

Comment (Qainguin) [reply src/TranslationUtils.gd]: Done!

**Comment by Qainguin** [comment]
You don't have to review by the way. I accidentally pressed the request review button.

**Comment by MewPurPur** [comment]
Thanks!

**Pull request merged**